15/umbraco-cms/reference/routing/iisrewriterules.md

```diff
@@ -83,11 +83,23 @@ For more details and other examples, take a look at the [URL Rewriting Middlewar
 
 ## Examples of rewrite rules
 
-* A great site showing 10 very handy IIS Rewrite rules: [URL rewriting tips and tricks](https://ruslany.net/2009/04/10-url-rewriting-tips-and-tricks/)
-* Another site showing some handy examples of IIS Rewrite rules: [Some useful IIS rewrite rules](https://odetocode.com/blogs/scott/archive/2014/03/27/some-useful-iis-rewrite-rules.aspx)
-* If you needed to a lot of static rewrites using rewrite maps: [Rule with rewrite map rule template](https://www.iis.net/learn/extensions/url-rewrite-module/rule-with-rewrite-map-rule-template)
+Below are some handy IIS Rewrite Rules you can use in your projects:
 
-For example, to always remove a trailing slash from the URL (make sure Umbraco doesn't add a trailing slash to all generated URLs by setting `AddTrailingSlash` to `false` in your [RequestHandler settings](../configuration/requesthandlersettings.md)):
+* **10 Handy IIS Rewrite Rules**
+
+A great resource showcasing 10 practical IIS Rewrite rules: [URL rewriting tips and tricks](https://ruslany.net/2009/04/10-url-rewriting-tips-and-tricks/)
+
+* **Examples of IIS Rewrite Rules**
+
+Another useful collection of IIS rewrite rule examples:  [Some useful IIS rewrite rules](https://odetocode.com/blogs/scott/archive/2014/03/27/some-useful-iis-rewrite-rules.aspx)
+
+* **Static Rewrites Using Rewrite Maps**
```

Comment:
I don't really understand why this is changed? 🤔 
It looks a bit strange in the preview:

![image](https://github.com/user-attachments/assets/cb652755-6ef0-463c-85b9-2b1311a073a4)

Reply:
It's interesting how it doesn't look like the way I expected it. 

I noticed that the article starts with headings followed by text, and then shifts to just text with code samples at the bottom. To maintain the flow I changed it this way. However, if you prefer, I can leave it as it is and just add headings to the code snippets below.

Reply:
I think the list looks better at least.
I do see your point about the consistency.. Would it make sense to give the list a little sub heading as well - something like "External examples" or "External resources"? 🤔 
And then a little sentence or two, about what the links are for? 🤔

Reply:
I've added a sub-heading and a 1 liner description to it. Let me know what you think.